pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php

```diff
@@ -361,6 +361,19 @@ public function additional_meta_box_fields( $order_id, $is_disabled, $dhl_label_
 					echo '<div class="shipment-dhl-icon-container"><span class="shipment-dhl-icon shipment-dhl-icon-crossborder-domestic"></span> ' . __( 'Crossborder', 'dhl-for-woocommerce' ) . '</div>';
 
 
+                    // PDDP
+                    if( $this->is_UK_shipment( $order_id ) ) {
+	                    $PDDP_value = $dhl_label_items['pr_dhl_PDDP'] ?? '';
+	                    woocommerce_wp_checkbox( array(
+		                    'id'          		=> 'pr_dhl_PDDP',
+		                    'label'       		=> __( 'POSTAL DELIVERED DUTY PAID: ', 'dhl-for-woocommerce' ),
+		                    'placeholder' 		=> '',
+		                    'description'		=> '',
+		                    'value'       		=> $PDDP_value,
+		                    'custom_attributes'	=> array( $is_disabled => $is_disabled )
+	                    ) );
+                    }

```

Comment:
```suggestion
                    if ( $this->is_UK_shipment( $order_id ) ) {
	                    $PDDP_value = $dhl_label_items['pr_dhl_PDDP'] ?? '';
	                    woocommerce_wp_checkbox( array(
		                    'id'                             => 'pr_dhl_PDDP',
		                    'label'                        => esc_html__( 'POSTAL DELIVERED DUTY PAID: ', 'dhl-for-woocommerce' ),
		                    'placeholder'            => '',
		                    'description'             => '',
		                    'value'                       => esc_attr( $PDDP_value ),
		                    'custom_attributes' => array( $is_disabled => $is_disabled )
	                    ) );
                    }

```
just a little suggestion for escaping the value and the label.